src/stylesheets/components/_site-alert.scss

```diff
@@ -0,0 +1,61 @@
+// Alert variables ---------- //
+// TODO: Custom alerts are effectively hidden and not well supported
+// Consider removing or rebuilding
```

Comment:
I don't know what functionality this is meant to be placeholding

Reply:
This is also in alert.scss as well. https://github.com/uswds/uswds/blob/fae131fd45e60f3545537415be12c0e7fd005771/src/stylesheets/components/_alerts.scss

@Dano, also, would you see a way to merge the two `$usa-alerts`s so that there was no duplication?